Winget-AutoUpdate/functions/Test-Network.ps1

```diff
@@ -1,82 +1,77 @@
-#Function to check the connectivity
-
-function Test-Network {
-
-    #Init
-    $timeout = 0
-
-    #Test connectivity during 30 min then timeout
-    Write-ToLog "Checking internet connection..." "Yellow"
-    While ($timeout -lt 1800) {
-
-        $URLtoTest = "https://raw.githubusercontent.com/Romanitho/Winget-AutoUpdate/main/LICENSE"
-        $URLcontent = ((Invoke-WebRequest -URI $URLtoTest -UseBasicParsing).content)
-
-        if ($URLcontent -like "*MIT License*") {
-
-            Write-ToLog "Connected !" "Green"
-
-            #Check for metered connection
-            [void][Windows.Networking.Connectivity.NetworkInformation, Windows, ContentType = WindowsRuntime]
-            $cost = [Windows.Networking.Connectivity.NetworkInformation]::GetInternetConnectionProfile().GetConnectionCost()
-
-            if ($cost.ApproachingDataLimit -or $cost.OverDataLimit -or $cost.Roaming -or $cost.BackgroundDataUsageRestricted -or ($cost.NetworkCostType -ne "Unrestricted")) {
-
-                Write-ToLog "Metered connection detected." "Yellow"
-
-                if ($WAUConfig.WAU_DoNotRunOnMetered -eq 1) {
-
-                    Write-ToLog "WAU is configured to bypass update checking on metered connection"
-                    return $false
-
-                }
-                else {
-
-                    Write-ToLog "WAU is configured to force update checking on metered connection"
-                    return $true
-
-                }
-
-            }
-            else {
-
-                return $true
-
+# Function to check the connectivity
+
+function Test-Network
+{
+   #Init
+
+   $timeout = 0
+
+   # Test connectivity during 30 min then timeout
+   Write-ToLog -LogMsg 'Checking internet connection...' -LogColor 'Yellow'
+   while ($timeout -lt 1800)
+   {
+      $URLtoTest = 'https://raw.githubusercontent.com/Romanitho/Winget-AutoUpdate/main/LICENSE'
+      $URLcontent = ((Invoke-WebRequest -Uri $URLtoTest -UseBasicParsing).content)
+
+      if ($URLcontent -like '*MIT License*')
+      {
+         Write-ToLog -LogMsg 'Connected !' -LogColor 'Green'
+
+         # Check for metered connection
+         $null = (Add-Type -AssemblyName Windows.Networking)
```

Comment:
What is this for? It generates errors when running manually:
![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/96626929/7603ff9b-db69-4cfb-bc4a-78440687f281)

Reply:
@Romanitho, that might be UWP related 
[windows.networking @ microsoft](https://learn.microsoft.com/en-us/uwp/api/windows.networking?view=winrt-22000)

Reply:
With `you` you're referring to @jhochwald

Reply:
This can safely be removed! Normally adding the assembly, before using it, is what I would call „good practice“. In this case I was wrong! With newer Windows PowerShell versions (and with core) adding a basic assembly causes issues.

Reply:
And as a note to myself, I should to the regression testing on a plain vanilla VM!
I did it on m highly customized dev box. Not the best idea I figured

Reply:
I saw other assemblies added. Should we clean these also?

Reply:
To prevent any further issues: I would recommend to remove them.
Normally, PowerShell handles that well by itself.

Reply:
Could you please make the changes regarding your tests on your 'vanilla VM'? :)

Reply:
`Winget-AutoUpdate-Install.ps1` (Line 511):
```
      #Log file and symlink initialization
      . "$WingetUpdatePath\functions\Start-Init.ps1"
      Start-Init
```
Not necessary to run Function `Start-Init` it twice... ...blame the commit?